src/stores/appcatalog/actions.ts

```diff
@@ -203,13 +203,23 @@ export function catalogLoadIndex(
 async function loadIndexForCatalog(catalog: IAppCatalog): Promise<IAppCatalog> {
   let indexURL = `${catalog.spec.storage.URL}index.yaml`;
 
+  // If we are trying to reach the Helm Stable catalog at it's old location,
+  // route the request through a proxy in Happa's container which adds necessary
+  // CORS headers.
```

Comment:
This comment is a boyscout commit to clarify and technically unrelated to this PR.